tests/skr/proxy.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import base64
+import json
+import subprocess
+from flask import Flask, request, Response
+import requests
+
+app = Flask(__name__)
```

Comment:
Why the switch to Flask from the http lib? We used Flask with the Hello World ACI demo but ended up having problems with Flask in conf AKS, so we switched to NGINX for the Hello World AKS demo. I don't think it matters for this test, but curious as to why the switch from the ACI Examples repo tests.

Reply:
No special reason other than I found Flask a little more concise to express the endpoints

Reply:
As far as testing goes, we could utilize flask to actually make an html webpage where we output debug info so that we don't have to exec into the container (adding the --debug-mode flag to the policy) each time it fails.

Reply:
We can, `c_aci_testing` also helps in this use case by printing out all container logs on cleanup